modules/ROOT/pages/configuration/dynamic-settings.adoc

```diff
@@ -1,20 +1,23 @@
 [role=enterprise-edition]
 [[dynamic-settings]]
 = Dynamic settings
-:description: How to change your Neo4j configuration while Neo4j is running, and which settings can be changed. 
+:description: How to change your Neo4j configuration while Neo4j is running, and which settings can be changed.
 
 
 [[dynamic-settings-introduction]]
 == Introduction
 
 Neo4j Enterprise Edition supports changing some configuration settings at runtime, without restarting the service.
+Changes to the configuration at runtime are not persisted.
+To avoid losing changes when restarting Neo4j, make sure you update xref:configuration/file-locations.adoc[_neo4j.conf_] as well.
```

Comment:
```suggestion
To avoid losing changes when restarting Neo4j, you must ensure that the xref:configuration/file-locations.adoc[_neo4j.conf_ file] is also updated.
```